backend/ResQueue/ResQueue/Endpoints/MessagesEndpoints.cs

```diff
@@ -25,6 +26,18 @@ public static void MapMessageEndpoints(this IEndpointRouteBuilder routes)
                     : Results.Problem(result.Problem!);
             });
 
+        group.MapGet("{transportMessageId}",
```

Comment:
Use {transportMessageId:guid} instead to avoid Postgres query errors. Postgres cannot compare strings and GUIDs directly, so the GUID type must be explicitly specified.

Also, update the GetSingleMessageRequest to use a Guid property for transportMessageId and so on. This way you won't need to modify Postgres query at all.

Make these changes to ensure the code is clear and works correctly with Postgres.

Reply:
fixed